### PR TITLE
[tests] fix makefile gator error

### PR DESCRIPTION
**Pull request by RomanenkoDenys**
Signed-off-by: Denis Romanenko <denis.romanenko@flant.com>

## Description

We have an error in main makefile, which prevents to download gator binary.

## Checklist
- [x] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: tests
type: fix
summary: fix makefile gator error.
impact_level: low
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -83,7 +83,7 @@ deps: bin/golangci-lint bin/trivy bin/regcopy bin/jq bin/yq bin/crane bin/promto
 
 bin/promtool-${PROMTOOL_VERSION}/promtool:
 	mkdir -p bin/promtool-${PROMTOOL_VERSION}
-	curl -sSfL https://github.com/prometheus/prometheus/releases/download/v${PROMTOOL_VERSION}/prometheus-${PROMTOOL_VERSION}.${GOHOSTOS}-${GOHOSTARCH}.tar.gz | tar zxf - -C bin/promtool-${PROMTOOL_VERSION} --strip=1 prometheus-${PROMTOOL_VERSION}.${GOHOSTOS}-${GOHOSTARCH}/promtool
+	curl -sSfL https://github.com/prometheus/prometheus/releases/download/v${PROMTOOL_VERSION}/prometheus-${PROMTOOL_VERSION}.${GOHOSTOS}-${GOHOSTARCH}.tar.gz -o - | tar zxf - -C bin/promtool-${PROMTOOL_VERSION} --strip=1 prometheus-${PROMTOOL_VERSION}.${GOHOSTOS}-${GOHOSTARCH}/promtool
```
Comment (gmaiorov) [inline Makefile:86]: `curl` outputs to stdout by default, explicit option is unneccesary.

File: Makefile [hunk 2]
```diff
@@ -92,7 +92,7 @@ bin/promtool: bin/promtool-${PROMTOOL_VERSION}/promtool
 
 bin/gator-${GATOR_VERSION}/gator:
 	mkdir -p bin/gator-${GATOR_VERSION}
-	curl -sSfL https://github.com/open-policy-agent/gatekeeper/releases/download/v${GATOR_VERSION}/gator-v${GATOR_VERSION}-${GOHOSTOS}-${GOHOSTARCH}.tar.gz | tar zxf - -C bin/gator-${GATOR_VERSION} --strip=1 gator
+	curl -sSfL https://github.com/open-policy-agent/gatekeeper/releases/download/v${GATOR_VERSION}/gator-v${GATOR_VERSION}-${GOHOSTOS}-${GOHOSTARCH}.tar.gz -o - | tar zxf - -C bin/gator-${GATOR_VERSION} gator
```
Comment (gmaiorov) [inline Makefile:95]: `curl` outputs to stdout by default, explicit option is unneccesary.
Here removed `--strip=1` is what fixes download, cause gator wasn't nested it a directory as promtool was.

File: Makefile (hunk 1 above)
Comment (RomanenkoDenys) [reply Makefile:86]: curl work differs on different distros. On my mac this was a problem.

File: Makefile (hunk 2 above)
Comment (RomanenkoDenys) [reply Makefile:95]: curl work differs on different distros. On my mac this was a problem.

**Review by gmaiorov — APPROVED** [review]
Tested changed makefile targets locally & using matrix tests, LGTM.

**Pull request merged**